### ISSUE TITLE
双系统的训练

### ISSUE THREAD

**Issue by markAKB** [InternRobotics/InternNav#262]
在训练时使用了traj_images提取到的特征，进行噪声的预测。而加噪轨迹基于traj_poses，两者在时间上是一一对应的。请问我的理解有问题吗？如果是这样，是不是会训练中会出现未来帧的信息泄露的问题？

以下是我阅读到的训练时forward和dataset加载的代码：
https://github.com/InternRobotics/InternNav/blob/0f822526d036cfa80f29865f8384e6f2140ca042/internnav/model/basemodel/internvla_n1/internvla_n1.py#L255

https://github.com/InternRobotics/InternNav/blob/0f822526d036cfa80f29865f8384e6f2140ca042/internnav/dataset/internvla_n1_lerobot_dataset.py#L1127

**Comment by markAKB** [comment]
> [InternNav/internnav/model/basemodel/internvla_n1/internvla_n1.py](https://github.com/InternRobotics/InternNav/blob/0f822526d036cfa80f29865f8384e6f2140ca042/internnav/model/basemodel/internvla_n1/internvla_n1.py#L239)
> 
> Line 239 in [0f82252](/InternRobotics/InternNav/commit/0f822526d036cfa80f29865f8384e6f2140ca042)
> 
>  images_dp = torch.stack([pix_goal_images, cur_images], dim=1).permute(0, 1, 4, 2, 3) 
> 
> 可以打印这里的size看下 设的bs是system 2的batch size 这里为了训到异步 system 1的batch size 是bs * future traj_len
> 对于每个sample来说 只有两张图 pixel goal image 和 cur image

明白了，很优雅的代码，感谢耐心解答。
